cortex-installer.sh

```diff
@@ -507,20 +507,20 @@ metadata:
   namespace: ${CORTEX_NAMESPACE}
   labels:
     app.kubernetes.io/name: spark-operator
-    app.kubernetes.io/version: v2.4.0-v1alpha1
```

Comment:
I think we should leave these as `v2.4.0-v1alpha1` until we update spark operator, since I think this refers to the version of the spark operator. When we update spark operator this will get updated too.